cluster-autoscaler/cloudprovider/aws/aws_util_test.go

```diff
@@ -0,0 +1,53 @@
+/*
+Copyright 2019 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package aws
+
+import (
+	"github.com/stretchr/testify/assert"
+	"strconv"
+	"testing"
+)
+
+func TestGetStaticEC2InstanceTypes(t *testing.T) {
+	result := GetStaticEC2InstanceTypes()
+	assert.True(t, len(result) != 0)
+}
+
+func TestParseMemory(t *testing.T) {
+	memoryInGiB := 3.75
+	expectedResultInMiB := int64(memoryInGiB * 1024)
+	result := parseMemory("3.75 GiB")
+	assert.Equal(t, expectedResultInMiB, result)
+
+	result = parseMemory("3.75 Gib")
+	assert.Equal(t, expectedResultInMiB, result)
+
+	result = parseMemory("3.75 Gib")
```

Comment:
AFAICT, this is the same test as on line 36.

Reply:
Also, for a future improvement, consider using table-based testing for things like this. Something like this would work:

```go
tests := []struct{
    input string
    expect float
}{
    {
        input: "3.75 GiB",
        expect: 3.75,
    },
    {
        input: "3.75 Gib",
        expect: 3.75,
    },
    {
        input: "3.75GiB",
        expect: 3.75,
    },
    {
        input: "3.75",
        expect: 3.75,
    },
}

for _, test := range tests {
    got := parseMemory(test.input)
    assert.Equal(t, test.expect, got)
}
```

Reply:
Address feedbacks in latest commit. Thanks!